specs/Transportation/TrafficFlowObserved/schema.json

```diff
@@ -74,6 +74,38 @@
             "backward"
           ]
         },
+        "vehicleType": {
+          "type": "string",
+          "enum": [
+            "agriculturalVehicle",
+            "bicycle",
+            "bus",
+            "minibus",
+            "car",
+            "tram",
+            "tanker",
+            "carWithCaravan",
+            "carWithTrailer",
+            "lorry",
+            "moped",
+            "motorcycle",
+            "motorcycleWithSideCar",
+            "motorscooter",
+            "trailer",
+            "van",
+            "caravan",
+            "constructionOrMaintenanceVehicle",
+            "trolley",
+            "binTrolley",
+            "sweepingMachine",
+            "cleaningTrolley"
+          ],
+          "metadata":{
```

Comment:
be careful, schemas only define attribute values not metadata. that's why I'm suggesting that you define `vehicleSubType` as a new attribute

Reply:
I was going on advice from synchronicity helpdesk with regards to using metadata to solve my problem. There is no fixed set of values for the vehicle sub type, so using metadata to specify a value seemed to make sense to me. I am open to advice from you about where we go, as I am fairly new to the fiware models and approach.

Reply:
metadata is for data that refers to an Attribute, such as timestamp, units, etc. in this case it is not so clear ... so that's why I'm advising to model it as a new Attribute

Reply:
will it need a specified set of values, similar to how the vehicleType does?